Sources/NimbleLoyalty.swift

```diff
@@ -105,6 +105,21 @@ extension NimbleLoyalty {
         }
     }
 
+    public func getRewardDetail(code: String, _ completion: @escaping (Result<APIReward, NimbleLoyaltyError>) -> Void) {
```

Comment:
@ducbm051291 You could update README about get reward detail in this PR

Reply:
I've already updated in this PR